packages/web/src/components/table/components/TableRepostButton.module.css

```diff
@@ -6,8 +6,8 @@
 }
 
 .icon {
-  width: 18px;
-  height: 18px;
+  width: 20px;
```

Comment:
we shouldn't need css sizing since harmony sizing is in effect

Reply:
these buttons aren't harmonized yet!

Reply:
ohhh I see, I thought these were related to the other icons you added. ty